apps/prairielearn/src/migrations/20230626165300_jobs__data__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE jobs
+ADD COLUMN data JSONB NOT NULL DEFAULT '{}'::JSONB;
```

Comment:
We could add a constraint to ensure that this is an object:
```
alter table books
add constraint books_doc_is_object
check (jsonb_typeof(doc) = 'object');
```
(From https://docs.yugabyte.com/preview/api/ysql/datatypes/type_json/create-indexes-check-constraints/ )

Reply:
Wow, neat! Done in ccf3fba.